init_dsp_duration : 1
ssp_duration : 0.9
dsp_duration : 0.1
final_dsp_duration : 1
step_height : 0.025
footsteps: #[x, y, z, yaw]
  [
    [0.0, 0.1, 0.0, 0.0],
    [0.0, -0.1, 0.0, 0.0],
    [0.2, 0.1, 0.0, 0.0],
    [0.4, -0.1, 0.0, 0.0],
    [0.6, 0.1, 0.0, 0.0],
    [0.8, -0.1, 0.0, 0.0],
    [1.0, 0.1, 0.0, 0.0],
    [1.2, -0.1, 0.0, 0.0],
    [1.4, 0.1, 0.0, 0.0],
    [1.4, -0.1, 0.0, 0.0]
  ]